tests/functional/object/mcg/test_noobaa_secret.py

```diff
@@ -355,3 +363,32 @@ def test_noobaa_root_secret():
         check_if_mcg_root_secret_public() is False
     ), "Seems like MCG root secrets are exposed publicly, please check"
     logger.info("MCG root secrets are not exposed to public")
+
+
+@mcg
+@red_squad
+@acceptance
+@bugzilla("2277186")
+@polarion_id("OCS-6184")
+def test_operator_logs_for_secret():
+    """
+    This test verifies if secrets are exposed
+    in noobaa operator logs
+
+    """
+
+    # get the noobaa operator logs
+    nb_operator_logs = get_pod_logs(pod_name=get_noobaa_operator_pod().name)
+    filtered_log = filter_pod_logs(
```

Comment:
https://github.com/red-hat-storage/ocs-ci/blob/e90b2edea045f472a52ceab4609860963699ead4/ocs_ci/helpers/helpers.py#L4562, same as my above comment. this existing func won't help?

Reply:
verify_log_exist_in_pods_logs() only checks for a single piece of log in whole log. It cant look for pattern of log. With filter_pod_logs, i will be able to filter the pod logs by certain pattern of logs.